vars/utils.groovy

```diff
@@ -504,11 +507,37 @@ def runLint() {
     }
 }
 
+def runCodeCov() {
 
+    node {
+          checkout scm
+          def tuna_docker = docker.build("ci-tuna:${branch_id}", "--build-arg FIN_TOKEN=${FIN_TOKEN} .")
+          tuna_docker.inside("--network host  --dns 8.8.8.8 ") {
+          env.TUNA_DB_HOSTNAME = "${db_host}"
+          env.TUNA_DB_NAME="${db_name}"
+          env.TUNA_DB_USER_NAME="${db_user}"
+          env.TUNA_DB_PASSWORD="${db_password}"
+          env.gateway_ip = "${gateway_ip}"
+          env.gateway_port = "${gateway_port}"
+          env.gateway_user = "${gateway_user}"
+          env.PYTHONPATH=env.WORKSPACE
+          env.PATH="${env.WORKSPACE}/tuna:${env.PATH}"
+          buildSchema()
+
+          sh "python3 -m coverage run -m pytest"
+          sh "python3 -m coverage json"
+          sh "mv coverage.json ../MITunaX/tests/covscripts/buffer"
+          sh "python3 tests/covscripts/parse_attributes.py"
+          sh "file= ../MITunaX/tests/covscripts/buffer/coverage_percentage.txt"
+          myvar = sh "\$(cat "$file")"
+          CODE_COV = 10
+          if (CODE_COV > myvar) {
+          error "Not added to env: ${item}"
+          }
```

Comment:
this seems to be commented out in the JK file. if its not called at all, then remove